openproblems/tasks/chromatin_potential/README.md

```diff
@@ -0,0 +1,18 @@
+# Chromatin accessibility prediction
+
+Chromatin accessibility prediction refers to the gene expression prediction of a cell or cell type from ATAC-seq peaks. For a summary or all relevant models, see gene score method in [Jeffrey M. Granja et al.](https://www.biorxiv.org/content/10.1101/2020.04.28.066498v1), [Su Wang et al.](https://pubmed.ncbi.nlm.nih.gov/24263090/) et al.
+
+## API
+
+Datasets should contain the following attributes:
```

Comment:
`checks.py` assumes existence of attributes not mentioned in your API.
```
    assert "mode2_var_chr" in adata.uns
    assert "mode2_var_start" in adata.uns
    assert "mode2_var_end" in adata.uns
```

Reply:
I added the attributes into API.